WrightTools/artists/_colors.py

```diff
@@ -133,6 +134,158 @@ def make_colormap(seq, name="CustomMap", plot=False):
     return cmap
 
 
+def rotate_colormap(cmap, rotations, eps=0.0001):
+    """
+    From a LinearSegmentedColormap, rotate an integer number of times, and return
+    a new colormap.   Note:  these rotations are limited by the use of epsilon
+    and may cause errors if the number of rotations is very large.  The last index value
+    will be off by eps x rotations.   The method only works on tuples and arrays for segmentdata;
+    use the r parameter instead for make_cubehelix.   Not used for ListedColormaps.
+
+    Parameters
+    ----------
+    cmap : matplotlib.colors.LinearSegmentedColormap
```

Comment:
I think you should make this work for any colormap, not just this specific type.